ui-performance.md

```diff
@@ -0,0 +1,470 @@
+---
+layout: page
+title: Jank No More
+subtitle: Where to look when your Flutter app has UI performance issues.
+description: Diagnosing UI performance issues in Flutter.
+permalink: /ui-performance/
+---
+
+<div class="whats-the-point" markdown="1">
+
+<b> <a id="whats-the-point" class="anchor" href="#whats-the-point" aria-hidden="true"><span class="octicon octicon-link"></span></a>What you'll learn:</b>
+
+* Flutter aims to provide 60 frames per second (fps) performance, and 120
+  fps performance on devices capable of 120Hz updates.
+* Frames should render in 16ms.
+* Jank occurs when the UI doesn't render smoothly. In other words, a
+  frame takes longer than 16ms to build and render.
+
+</div>
+
+This guide shows [PENDING: text].
+
+* TOC Placeholder
+{:toc}
+
+## A workflow for diagnosing jank
+
+<div class="whats-the-point" markdown="1">
+
+<b> <a id="whats-the-point" class="anchor" href="#whats-the-point" aria-hidden="true"><span class="octicon octicon-link"></span></a>What's the point?</b>
+
+* Profile your app on a physical device.
+* Profile your app in profile mode.
+* Check performance on the slowest device your users might use.
+* Start by enabling the performance overlay.
+
+</div>
+
+### Connect to a physical device
+
+Almost all performance debugging for Flutter applications should be
+conducted on a physical Android or iOS device, with your Flutter application
+running in profile mode. Using debug mode, or running apps on simulators
```

Comment:
I would put profile and debug modes in backquotes since they refer to a very specific thing.